en: Remove duplicate example

merkata opened the pull request:
With #16974 and the removal of --include-uninitialized flag, the second
and third examples of kubectl delete become equal, thus leading to
duplication and being confusing. Suggest to remove the duplicate and
replace it with another example in the future if needed.

Observed in v1.16 and v1.17 documentation.

Signed-off-by: Mariyan Dimitrov <mariyan.dimitrov@gmail.com>

merkata commented:
/assign @kbhawkey

merkata commented:
/check-cla

merkata commented:
I signed it

merkata commented:
/check-cla

php-coder commented:
@merkata The signature might not get recognized because the e-mail in sign-off and author-s email don't match (`mariyan.dimitrov+github@gmail.com` vs  `mariyan.dimitrov@gmail.com`).

merkata commented:
> @merkata The signature might not get recognized because the e-mail in sign-off and author-s email don't match (`mariyan.dimitrov+github@...` vs `mariyan.dimitrov@...`).

@php-coder yup, that is the case, I changed that, but I think I have one commit without the +github appended and one with, so the bot can't verify both commits :/
I've changed the e-mail with cncf to not include the +github and will re-run the check, here's hoping it works.

merkata commented:
/check-cla

merkata commented:
I've opened a new one #18656 so closing this PR, sorry for the mess...

php-coder commented:
Duplicate of #18656
